common/app/templates/inlineJS/blocking/curlConfig.scala.js

```diff
@@ -43,7 +43,8 @@ window.curlConfig = {
 
             // plugins
             text:         'components/requirejs-text/text',
-            inlineSvg:    'projects/common/utils/inlineSvg'
+            inlineSvg:    'projects/common/utils/inlineSvg',
+            ldsh:         'projects/common/utils/lodash-loader'
```

Comment:
Maybe this plugin should be called `template!` instead? Lodash is just the implementation of it.